stream-transformation-test/stream-transformation-it/pom.xml

```diff
@@ -67,6 +67,20 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <dependency>
+            <groupId>uk.gov.justice.event-store</groupId>
+            <artifactId>event-publisher-process</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>uk.gov.justice.event-store</groupId>
+            <artifactId>linked-event-processor</artifactId>
+            <version>${event-store.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>uk.gov.justice.event-store</groupId>
+            <artifactId>event-repository-jdbc</artifactId>
+            <version>${event-store.version}</version>
+        </dependency>
```

Comment:
Non test dependencies above should be moved above the test dependency

Reply:
done